jekyll/_docs/build-image-trusty.md

```diff
@@ -6,8 +6,9 @@ description: Ubuntu 14.04 (Trusty)
 changefreq: "weekly"
 ---
 
-You can run your Linux builds on Ubuntu 14.04 Trusty (the default). You can 
-switch to Trusty from "Project Settings" -> "Build Environment" of your project.
+You can run your Linux builds on the default image, Ubuntu 14.04 (Trusty). You 
+can switch to Trusty from "Project Settings" -> "Build Environment" of your 
+project.
```

Comment:
Some people may not longer be on Ubuntu 14.04 and want to switch back.

Some people who had an existing project before the switch was made, are still using Ubuntu 12.04 because that was the default. Now they may want to switch to the new default image.

The goal of this PR is to simply remove the reference of Ubuntu 12.04 being the default since that is now false.